ansible_navigator/configuration/application_post_processor.py

```diff
@@ -0,0 +1,181 @@
+""" post processing of ansible-navigator configuration
+"""
+import os
+
+from typing import Any
+from typing import List
+from typing import Tuple
+
+from .definitions import Entry
+from .definitions import EntrySource
+from .definitions import Config
+from .definitions import Message
+
+from ..utils import Sentinel
+
+
+def _post_processor(func):
+    def wrapper(*args, **kwargs):
+        name = kwargs["entry"].name
+        before = str(kwargs["entry"].value.current)
+        messages, errors = func(*args, **kwargs)
+        after = str(kwargs["entry"].value.current)
+        msg = f"Completed post processing for {name} before: '{before}' after: '{after}'"
+        messages.append(("debug", msg))
+        return messages, errors
+
+    return wrapper
+
+
+class ApplicationPostProcessor:
+    """application post processor"""
+
+    @staticmethod
+    def _abs_user_path(fpath):
+        return os.path.abspath(os.path.expanduser(fpath))
+
+    @staticmethod
+    def _str2bool(value: Any) -> bool:
```

Comment:
Function can be moved to utils file for common handling of boolean value across the App's